src/metax_api/services/auth_service.py

```diff
@@ -27,16 +27,20 @@ def get_user_projects(request):
         try:
             with open(settings.ADDITIONAL_USER_PROJECTS_PATH, 'r') as file:
                 file_projects = json.load(file)
-        except:
-            _logger.info("Unable to read projects from file")
+        except Exception as e:
+            if isinstance(e, FileNotFoundError):
```

Comment:
Just one more nitpick! In Python, the proper way to react to different types of exceptions would be:

```
except FileNotFoundError as e:
    # do something when file was not found
except OtherErrorType as e:
    # do something else
except Exception as e:
    # handling for all other types
```
(note that the `as e` part is optional: only needed when the code needs to get details out of the error, such as getting the exact error message, error code, etc...)